src/controllers/controller.js

```diff
@@ -14,14 +14,17 @@ admin.initializeApp({
 //Database reference
 const db = admin.firestore();
 
-function createDeveloper(name, status, rank) {
+function createDeveloper(name, status, rank, vacationDays) {
```

Comment:
Skal man angive vacationsdays ved oprettelse af en dev? :C

Reply:
Den er ikke required, så den kan bare være null/undefined

Reply:
Og hvis du kalder metoden `createDeveloper('myname', 'somestatus', 'pleb') så bliver vacationDays automatisk sat til undefined.

Reply:
Cool det anede jeg ikke, så man kan overloade som man har lyst til?

Reply:
Kinda, du kan ikke skippe de tidligere attributter, for så er rækkefølgen forkert.

Reply:
Sure giver mening